schema/project-level/project-schema.json

```diff
@@ -180,8 +180,26 @@
   "definitions": {
     "ContractingProcess": {
       "title": "Contracting process",
-      "description": "Within the project level data specification, a contracting process provides both summary information, and a log of changes over time, either manually curated, or automatically generated through linked OCDS releases. Where OCDS data is available, most ContractingProcess summary fields can be derived from OCDS releases, although the exact method to derive data may vary between implementations.",
+      "description": "Within the project level data specification, a contracting process provides both summary information, and a log of changes over time, either manually curated, or automatically generated through linked OCDS releases.",
       "type": "object",
+      "properties": {
+        "summary": {
+          "$ref": "#/definitions/Summary"
```

Comment:
A definition named `ContractingProcessSummary` might be preferred, in case other types of summaries are introduced in future.